documentation/book/snip-kafka-versions.adoc

```diff
@@ -0,0 +1,19 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-upgrading-kafka-versions.adoc
+
+// Generated by documentation/supported-version.sh during the build
+// DO NOT EDIT BY HAND
+
+[id='ref-kafka-versions-{context}']
+= Kafka versions
+
+[options="header"]
+|=================
+|Kafka version |Interbroker protocol version |Log message format version| Zookeeper version
+|2.0.0 |2.0 |2.0 |3.4.9
```

Comment:
Zookeeper version is IMHO not `3.4.9` but `3.4.13` for all of these versions.

Reply:
Then it's also wrong in the Kafka upstream: https://kafka.apache.org/documentation/#zkversion :open_mouth:

Reply:
Indeed it is wrong there as well. The binary downloads and Gradle builds do not lie (https://github.com/apache/kafka/blob/3402a8361b7347326b8a9fdb22300986fbbf9cd0/gradle/dependencies.gradle#L85). Will you open a Kafka PR? Or should I? ;-)

Reply:
Can you?